compact_str/src/lib.rs

```diff
@@ -2130,6 +2130,98 @@ impl<'a> From<&'a CompactString> for Cow<'a, str> {
     }
 }
 
+#[cfg(target_has_atomic = "ptr")]
```

Comment:
Can you annotate this with `#[rustversion(since(1.60))]`? This way we can support more architectures and include this impl?